nss-capricious-finches/nutshell
# Events PR

# Link to Ticket
Closes # Link

# Description of Proposed Changes
Steps to Test
git fetch --all
checkout the branch
load the page with an ID logged in and the events for that person should display
you have the option to add an event at the bottom and that should show in the data json
bottom left of the event box has a delete event button that should prompt to delete and then clear from the DOM and from the json
Outline the steps to test

```
git fetch --all
git checkout JR_Events
```

# Impacted Areas in Application
List general components of the application that this PR will affect:
Creating and displaying events to the DOM.
I created seperate modules for displaying and creating the events

# Mentions @username
Tag users that need to review this code

# Definition of Done
#### The project must be fully documented. This includes the following:
  1) Complete README that documents the steps to install the code, how to install any dependencies, or system configuration needed.
  2) Every class must be documented with purpose, author, and methods.
  3) Every method must be documented with purpose and argument list - which itself must contain a short purpose for each argument.
  4) The project must be able to run fully, and without errors, on each teammate's system.
  5) Fulfills every requirement. Every line of code has been peer reviewed.
  6) For projects that require unit testing, core functionality must be identified and have at least one test for each.
